### PR TITLE
Minor fixups

### PR DESCRIPTION
**Pull request by rictic**
* the type of Object.assign will get stricter in TS 4.7
* specify tsbuildinfo file name in ts-transformers so that the tsbuildinfo file is placed in the right place and doesn't clash with other packages' location.
* ignore .wireit directories in vscode

### CODE REVIEW

File: lit-next.code-workspace [hunk 1]
```diff
@@ -62,7 +62,10 @@
     "debug.javascript.warnOnLongPrediction": false,
     // Regardless of the user's usual formatter, files
     // in the Lit repo must be formatted using prettier.
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "files.exclude": {
```
Comment (aomarks) [inline lit-next.code-workspace:66]: Doesn't it already ignore everything in gitignore?

Comment (rictic) [reply lit-next.code-workspace:66]: Not completely, e.g. wireit files fill the cmd+P menu

[the remark this answers is not in the document]
Comment (rictic) [reply lit-next.code-workspace:66]: [dupe comment]

File: packages/ts-transformers/tsconfig.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
 
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
```
Comment (aomarks) [inline packages/ts-transformers/tsconfig.json:21]: I think this is the default but explicit good

**Review by aomarks — APPROVED** [review]

File: packages/ts-transformers/tsconfig.json (hunk 2 above)
Comment (rictic) [reply packages/ts-transformers/tsconfig.json:21]: It was reading/writing from the wrong location on my machine, and this fixed it. 🤷

**Pull request merged**